app/templates/layouts/_error.html

```diff
@@ -14,9 +14,8 @@ <h1 class="saturn">Error {{status_code}}</h1>
         {%- endfor -%}
       </ul>
     {%- endif -%}
-
-
-    <p class="u-mb-no">If the problem continues please call <a class="info__link" href="tel:03001234931" aria-describedby="survey-help">0300 1234 931</a></p>
+         {% set ariadescibedby="survey-help" %}
+    <p class="u-mb-no">If the problem continues please call {% include 'partials/widgets/telephone_contact.html' %}</a></p>
```

Comment:
This loses the class from the `a` element. If we change to use a macro as shown above we could pass the class as another parameter to the macro.

Reply:
This was removed following a discussion with Ben Armstrong, who decided it didn't add to the page.